examples/trials/mnist-pytorch/config_adl.yml

```diff
@@ -0,0 +1,21 @@
+authorName: default
+experimentName: example_mnist_pytorch
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+
+logCollection: http
+trainingServicePlatform: adl
+
+searchSpacePath: search_space.json
+useAnnotation: false
+tuner:
+  builtinTunerName: TPE
+  classArgs:
+    optimize_mode: maximize
+
+trial:
+  image: {replace_to_your_image_tag}  # built by ./adl.Dockerfile
```

Comment:
does adl need to build a new docker image when submit a new trial job every time? how to set docker image registry?